build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -121,15 +121,22 @@
 
       <!-- Downloaded Installers + Archives -->
       <DownloadedSharedHostInstallerFileName Condition=" '$(InstallerExtension)' != '' ">dotnet-host-$(ProductMonikerRid).$(SharedHostVersion)$(InstallerExtension)</DownloadedSharedHostInstallerFileName>
+      <DownloadedSharedHostInstallerFileNameWithToken Condition=" '$(CoreSetupBlobAccessToken)' != '' ">$(DownloadedSharedHostInstallerFileName)?$(CoreSetupBlobAccessToken)</DownloadedSharedHostInstallerFileNameWithToken>
```

Comment:
Can you do this differently?

Instead of having this defined twice (and the others), maybe have a `CoreSetupBlobAccessTokenParam`. And then you can have CoreSetupBlobAccessToken equal to `'?$(CoreSetupBlocAccessTokenParam)'` when the Param is not empty. And then you just append the token to all URLs.